test/controllers/auth/AuthActionSpec.scala

```diff
@@ -101,30 +101,26 @@ class AuthActionSpec extends FreeSpec with MustMatchers with MockitoSugar with O
     }
   }
 
+  implicit class Ops[A](a: A) {
+    def ~[B](b: B): A ~ B = new ~(a, b)
+  }
+
   "A user with nino and no SA enrolment must" - {
     "create an authenticated request" in {
 
       val nino = Fixtures.fakeNino.nino
       val retrievalResult: Future[
         Option[String] ~ Enrolments ~ Option[Credentials] ~ ConfidenceLevel ~ Option[UserName] ~ LoginTimes ~ Option[
```

Comment:
This type is repeated a lot in this file and other files. Consider creating a type variable somewhere common so it can be referenced once and only needs to be changed once
```
type AuthRetrievals = Option[String] ~ Enrolments ~ Option[Credentials] ~ ConfidenceLevel ~ Option[UserName] ~ LoginTimes ~ Option[TrustedHelper]
```